cluster/gce/coreos/helper.sh

```diff
@@ -46,6 +46,11 @@ function create-master-instance() {
   local address_opt=""
   [[ -n ${1:-} ]] && address_opt="--address ${1}"
 
+  local preemptible_master=""
+  if [[ "${PREEMPTIBLE_MASTER}" == "true" ]]; then
```

Comment:
This is called from kube-up.sh which has `set -o nounset` at the top. So if you don't define $PREEMTPIBLE_MASTER then the shell script would exit if the variable is unset. But your change to config-default ensures that you are always setting it (to false by default) so unless that changes (or this function is called without sourcing that file) then the variable will always have a value.